Makefile

```diff
@@ -136,6 +136,28 @@ docker-run:
 	@docker-compose up --build
 	@echo "++\n*****"
 
+api-unit-test:
+	@echo "++\n***** Running API unit tests\n++"
+	@yarn workspace @ehpr/common build
+	@yarn workspace @ehpr/api build
+	@yarn workspace @ehpr/api test
+	@echo "++\n*****"
+
+start-local-db:
+	NODE_ENV=test docker-compose -f docker-compose.test.yaml up --build -d
+
+stop-local-db:
+	NODE_ENV=test docker-compose -f docker-compose.test.yaml down
+
+api-integration-test:
+	@echo "++\n***** Running API integration tests\n++"
+	@yarn workspace @ehpr/common build
+	@yarn workspace @ehpr/api build
+	@yarn workspace @ehpr/api test:e2e
+	@echo "++\n*****"
+
+api-integration-test-ci: start-local-db api-integration-test stop-local-db
```

Comment:
there could be a race condition, possibly we need some wait after db start.

Reply:
Nest/TypeORM seems to have retry/timeout capabilities from what I've seen in the current failures